Index: git/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,10 @@
 all: udooclient udooserver
 
 udooclient: udooclient.c
-	gcc -Wall udooclient.c -o udooclient
+	${CC} -Wall udooclient.c -o udooclient
 
 udooserver: udooserver.c
-	gcc -Wall udooserver.c -o udooserver
+	${CC} -Wall udooserver.c -o udooserver
 
 clean:
 	rm -rf udooclient udooserver
