Index: git/arch/arm/boot/dts/imx6qdl-seco_A62.dtsi
===================================================================
--- git.orig/arch/arm/boot/dts/imx6qdl-seco_A62.dtsi
+++ git/arch/arm/boot/dts/imx6qdl-seco_A62.dtsi
@@ -261,7 +261,7 @@
 				MX6QDL_PAD_NANDF_D3__GPIO2_IO03   	0x80000000      /* POWER ON */
 				MX6QDL_PAD_NANDF_WP_B__GPIO6_IO09 	0x80000000	/*GPIO*/
 				MX6QDL_PAD_GPIO_18__GPIO7_IO13	  	0x80000000	/*GPIO*/
-				MX6QDL_PAD_GPIO_19__GPIO4_IO05	  	0x80000000	/*GPIO*/				
+				MX6QDL_PAD_GPIO_19__GPIO4_IO05	  	0xE870   	/*GPIO pin 17 enable openDrain, see iomux tool to configure that*/				
 				//MX6QDL_PAD_GPIO_7__GPIO1_IO07	  	0x80000000	/*FLEXCAN1 TX*/
 				//MX6QDL_PAD_GPIO_8__GPIO1_IO08	  	0x80000000      /*FLEXCAN1 RX*/
 
@@ -273,7 +273,7 @@
 
 				MX6QDL_PAD_CSI0_DAT18__GPIO6_IO04 	0x1f071      /* Camera Power Enable */
 				MX6QDL_PAD_CSI0_DAT19__GPIO6_IO05 	0x1f071	/* Camera Reset */
-				MX6QDL_PAD_CSI0_MCLK__CCM_CLKO1			0x130b0		// CSI master clock (CN11)
+				MX6QDL_PAD_CSI0_MCLK__CCM_CLKO1		0x130b0		// CSI master clock (CN11)
 
 				MX6QDL_PAD_NANDF_CS0__GPIO6_IO11  	0x80000000    	/* SATA led activity */ 	
 				MX6QDL_PAD_EIM_A25__GPIO5_IO02    	0x80000000     	/* HDMI - check */
