Index: git/arch/arm/boot/dts/imx6qdl-seco_A62.dtsi
===================================================================
--- git.orig/arch/arm/boot/dts/imx6qdl-seco_A62.dtsi
+++ git/arch/arm/boot/dts/imx6qdl-seco_A62.dtsi
@@ -333,9 +333,16 @@
 			fsl,pins = <
 				MX6QDL_PAD_GPIO_5__I2C3_SCL             0x4001b8b1
 				MX6QDL_PAD_GPIO_6__I2C3_SDA             0x4001b8b1
-
-				MX6QDL_PAD_SD2_DAT2__GPIO1_IO13   	0x80000000       /*touch interrupt*/ 
-				MX6QDL_PAD_SD2_DAT0__GPIO1_IO15   	0x80000000       /* touch reset */ 
+			>;
+		};
+                
+/*  __________________________________________________________________________
+ * |________________________________ TOUCH ___________________________________|
+ */
+		pinctrl_st1232: st1232grp {
+			fsl,pins = <
+				MX6QDL_PAD_SD2_DAT2__GPIO1_IO13              0x80000000		// Touch Interrupt
+				MX6QDL_PAD_SD2_DAT0__GPIO1_IO15              0x80000000		// Touch Reset
 			>;
 		};
 /*  __________________________________________________________________________
@@ -653,10 +660,22 @@
 	touchscreen@55 {
  		compatible = "sitronix,st1232";
  		reg = <0x55>;
+                pinctrl-names    = "default";
+		pinctrl-0        = <&pinctrl_st1232>;
  		interrupt-parent = <&gpio1>;
  		interrupts = <13 IRQ_TYPE_LEVEL_LOW>;
  		gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
  	};
+        gt928@5d {
+		compatible       = "goodix,gt928";
+		reg              = <0x5d>;
+		pinctrl-names    = "default";
+		pinctrl-0        = <&pinctrl_st1232>;
+		interrupt-parent = <&gpio1>;
+		interrupts       = <13 IRQ_TYPE_LEVEL_LOW>;
+		irq-gpio         = <&gpio1 13 GPIO_ACTIVE_LOW>;
+		reset-gpio       = <&gpio1 15 GPIO_ACTIVE_LOW>;
+	};
 };
 /*  __________________________________________________________________________
  * |__________________________________________________________________________|
