Index: git/drivers/input/touchscreen/Kconfig
===================================================================
--- git.orig/drivers/input/touchscreen/Kconfig
+++ git/drivers/input/touchscreen/Kconfig
@@ -930,6 +930,17 @@ config TOUCHSCREEN_STMPE
 
 	  To compile this driver as a module, choose M here: the
 	  module will be called stmpe-ts.
+config TOUCHSCREEN_GOODIX
+        tristate "Goodix I2C touchscreen"
+        depends on I2C
+        depends on GPIOLIB || COMPILE_TEST
+        help
+          Say Y here if you have the Goodix touchscreen (such as one
+          installed in Onda v975w tablets) connected to your
+          system. It also supports 5-finger chip models, which can be
+          found on ARM tablets, like Wexler TAB7200 and MSI Primo73.
+
+          If unsure, say N.To compile this driver as a module, choose M here: the
 
 config TOUCHSCREEN_SUR40
 	tristate "Samsung SUR40 (Surface 2.0/PixelSense) touchscreen"
