Index: git/drivers/input/touchscreen/Makefile
===================================================================
--- git.orig/drivers/input/touchscreen/Makefile
+++ git/drivers/input/touchscreen/Makefile
@@ -31,9 +31,10 @@ obj-$(CONFIG_TOUCHSCREEN_HAMPSHIRE)	+= h
 obj-$(CONFIG_TOUCHSCREEN_GUNZE)		+= gunze.o
 obj-$(CONFIG_TOUCHSCREEN_EETI)		+= eeti_ts.o
 obj-$(CONFIG_TOUCHSCREEN_ELO)		+= elo.o
-obj-$(CONFIG_TOUCHSCREEN_ELAN)      += elan_ts.o
+obj-$(CONFIG_TOUCHSCREEN_ELAN)      	+= elan_ts.o
 obj-$(CONFIG_TOUCHSCREEN_EGALAX)	+= egalax_ts.o
 obj-$(CONFIG_TOUCHSCREEN_FUJITSU)	+= fujitsu_ts.o
+obj-$(CONFIG_TOUCHSCREEN_GOODIX) 	+= goodix.o
 obj-$(CONFIG_TOUCHSCREEN_ILI210X)	+= ili210x.o
 obj-$(CONFIG_TOUCHSCREEN_INEXIO)	+= inexio.o
 obj-$(CONFIG_TOUCHSCREEN_INTEL_MID)	+= intel-mid-touch.o
