Index: git/include/linux/device.h
===================================================================
--- git.orig/include/linux/device.h
+++ git/include/linux/device.h
@@ -609,6 +609,8 @@ extern int devres_release_group(struct d
 
 /* managed devm_k.alloc/kfree for device drivers */
 extern void *devm_kmalloc(struct device *dev, size_t size, gfp_t gfp);
+extern char *devm_kvasprintf(struct device *dev, gfp_t gfp, const char *fmt,va_list ap);
+extern char *devm_kasprintf(struct device *dev, gfp_t gfp,const char *fmt, ...);
 static inline void *devm_kzalloc(struct device *dev, size_t size, gfp_t gfp)
 {
 	return devm_kmalloc(dev, size, gfp | __GFP_ZERO);
