Index: git/drivers/tty/serial/imx.c
===================================================================
--- git.orig/drivers/tty/serial/imx.c	2016-05-12 15:30:47.840863698 +0200
+++ git/drivers/tty/serial/imx.c	2016-05-12 15:35:28.120858578 +0200
@@ -1221,7 +1221,8 @@
 
 	/* Can we enable the DMA support? */
 	if (is_imx6q_uart(sport) && !uart_console(port)
-		&& !sport->dma_is_inited)
+		&& !sport->dma_is_inited
+		&& !sport->port.flags == UPF_LOW_LATENCY)
 		imx_uart_dma_init(sport);
 
 	if (sport->dma_is_inited) {
